.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'State: Unconfirmed, Type: Bug'
+assignees: m273d15 
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Log files**
+
+Please provide the corresponding log files, but be sure to **remove or obfuscate sensitive data** you don't want to publish.
+
+You can find the log files in the following locations:
+* IntelliJ IDEA (with default config), where `.IdeaXXXXXXXX` is the directory containing the IDE files (e.g. `.IdeaIC2019.1`).
+  More information about the location of the directory is given here: https://intellij-support.jetbrains.com/hc/en-us/articles/206544519-Directories-used-by-the-IDE-to-store-settings-caches-plugins-and-logs
+  * IDE logs - `~/.IdeaXXXXXXXX/system/log/`
+  * Saros logs - `~/.IdeaXXXXXXXX/system/log/SarosLogs/`
+* Eclipse:
+  * IDE logs - `<workspace>/.metadata/.log`
```

Comment:
is the file really named `.log` or should this be `*.log` as well?

Reply:
Yes, the name `.log` is correct